services/brig/src/Brig/User/Search/TeamUserSearch.hs

```diff
@@ -102,12 +102,19 @@ teamUserSearchQuery tid mbSearchText _mRoleFilter mSortBy mSortOrder =
         mbQStr
     )
     teamFilter
-    ( maybe
+    -- todo(leif): add comment why this is necessary
+    (defaultSorts ++ [sortingTieBreaker])
+  where
+    defaultSorts :: [ES.DefaultSort]
+    defaultSorts =
+      maybe
         [defaultSort SortByCreatedAt SortOrderDesc | isNothing mbQStr]
         (\tuSortBy -> [defaultSort tuSortBy (fromMaybe SortOrderAsc mSortOrder)])
         mSortBy
-    )
-  where
+
+    sortingTieBreaker :: ES.DefaultSort
```

Comment:
extreme nitpick: why not `:: [ES.DefaultSort]`?

Reply:
Nitpick or not, you've got a point!